.gitignore

```diff
@@ -23,3 +23,10 @@ selftest
 src/static.c
 build*
 *.pc
+
+# Backup files
+*~
+*.bak
+*.old
+*.orig
+*.rej
```

Comment:
nit: this can be a separate PR

Reply:
That got in by accident, I'll remove it.